Content.Server/Traits/TraitSystem.Functions.cs

```diff
@@ -698,3 +701,58 @@ public override void OnPlayerSpawn(EntityUid uid,
         tagSystem.AddTags(uid, Tags);
     }
 }
+
+// <summary>
+//      Replaces a body part with a cybernetic. This is only for limbs such as arms and legs, don't use this for organs(old or new).
+// </summary>
+[UsedImplicitly]
+public sealed partial class TraitCyberneticLimbReplacement : TraitFunction
+{
+    [DataField, AlwaysPushInheritance]
+    public BodyPartType RemoveBodyPart { get; private set; } = BodyPartType.Arm;
+
+    [DataField, AlwaysPushInheritance]
+    public BodyPartSymmetry PartSymmetry { get; private set; } = BodyPartSymmetry.Left;
+
+    [DataField, AlwaysPushInheritance]
+    public EntProtoId? ProtoId { get; private set; }
+
+    [DataField, AlwaysPushInheritance]
+    public string SlotId { get; private set; } = "right arm";
+
+    public override void OnPlayerSpawn(EntityUid uid,
+        IComponentFactory factory,
+        IEntityManager entityManager,
+        ISerializationManager serializationManager)
+    {
+        var bodySystem = entityManager.System<BodySystem>();
+        var transformSystem = entityManager.System<SharedTransformSystem>();
+
+        if (!entityManager.TryGetComponent(uid, out BodyComponent? body)
+            || !entityManager.TryGetComponent(uid, out TransformComponent? xform)
```

Comment:
```suggestion
            || !entityManager.TryGetComponent(uid, out TransformComponent? transform)
```
This doesn't seem like a clear abbreviation--code is _read_ more often than it is written, so abbreviating like this might not be helpful for other contributed seeking to abuse this system later

Reply:
isn't xform standard for the transformcomponent?

Reply:
Yea, I was about to say this is actually a standard abbreviation for this particular example.